frontend/drivers/platform_ps2.c

```diff
@@ -332,6 +323,14 @@ static void frontend_ps2_init(void *data)
    audsrv_init();
 
    SDL_Init(SDL_INIT_TIMER);
+
+
+retro_sleep(100);
```

Comment:
Is this sleep needed?

Reply:
I need to improve it, so far I have noticed that a delay is needed in order to have the USB ready. I saw PS2 project and they use it as well.
Anyway, this is still an in-progress port, I'm just creating small PRs in order to make easier the review.

If you want I can remove it, but I can not assure if it will always work with USBs.